sigstore/errors.py

```diff
@@ -26,9 +27,9 @@ class Error(Exception):
     def diagnostics(self) -> str:
         """Returns human-friendly error information."""
 
-        return """An issue occurred."""
+        return str(self)
 
-    def print_and_exit(self, raise_error: bool = False) -> None:
+    def log_and_exit(self, logger: Logger, raise_error: bool = False) -> None:
```

Comment:
this is maybe unrelated: is `raise_error` useful for anything?

Reply:
So-so: we use it to spit out the original traceback when `args.verbose >= 1`, but I could be convinced to just inline that logic directly into `_cli.py`'s error handling.

Reply:
(I'll put this in the refactoring issue.)